sdk/common/src/main/java/io/opentelemetry/sdk/common/InstrumentationScopeInfo.java

```diff
@@ -26,15 +27,25 @@ public abstract class InstrumentationScopeInfo {
 
   /** Creates a new instance of {@link InstrumentationScopeInfo}. */
   public static InstrumentationScopeInfo create(String name) {
-    requireNonNull(name, "name");
-    return new AutoValue_InstrumentationScopeInfo(name, null, null);
+    return InstrumentationScopeInfo.create(name, null, null, Attributes.empty());
   }
 
   /** Creates a new instance of {@link InstrumentationScopeInfo}. */
   public static InstrumentationScopeInfo create(
       String name, @Nullable String version, @Nullable String schemaUrl) {
+    return InstrumentationScopeInfo.create(name, version, schemaUrl, Attributes.empty());
+  }
+
+  /**
+   * Creates a new instance of {@link InstrumentationScopeInfo}.
+   *
+   * @since 1.18.0
+   */
+  public static InstrumentationScopeInfo create(
+      String name, @Nullable String version, @Nullable String schemaUrl, Attributes attributes) {
```

Comment:
we might be at the point where we introduce a builder for this fella

Reply:
For me it was a question of likelihood that additional fields are added to instrumentation scope in the future. I think its unlikely, but it could still happen so probably best to introduce a builder. 

How about I add a builder and deprecate the the methods that accept version and schema?